### DISCUSSION TITLE
Configuration, warmup: 'none', 'face', 'full'

### DISCUSSION THREAD

**Discussion by meeki007** [vladmandic/human#73] [Q&A] [answered]
As per the documentation --- > https://github.com/vladmandic/human/wiki/Configuration

what to use for human.warmup(), can be 'none', 'face', 'full
warmup pre-initializes all models for faster inference but can take significant time on startup

Question:
If I just want to use body and I have set body to
```
enabled: true
```

And Ive set face and hand to:
```
enabled: false
```

then I set:
```
warmup : full
```

will skip face and hand for the warmup?
or can I set :
```
warmup : body
```

writing user selection logic and want to make sure i get it right.

thank you,
meeki

**Comment by vladmandic** [comment, marked as answer]
`warmup: body` or `warmup: face` simply selects image used for warmup, it doesn't change the `human` configuration.  
so warmup will follow any user provided configuration rules regardless of the warmup image selected.
